spec/features/saml/loa1_sso_spec.rb

```diff
@@ -81,7 +81,7 @@
     it 'user can view and confirm personal key during sign up', :js do
       allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
       user = create(:user, :with_phone)
-      code = 'ABC1-DEF2-GHI3-JKL4'
+      code = 'ABC1-DEF2-GH13-JK14'
```

Comment:
What was the reason for changing this?

Reply:
This example key was incorrect based on our code, and wasn't exposed earlier for lack of test coverage. See:

https://github.com/18F/identity-idp/blob/master/app/services/personal_key_generator.rb#L35